src/status_im/data_store/realm/core.cljs

```diff
@@ -84,11 +84,26 @@
 (defn write [realm f]
   (.write realm f))
 
+(defn- prepare-for-transit 
+  "Following Transit documentation, escape leading ~'s
+  in strings by prepending a ~. This prepares for subsequent 
+  fetching from Realm where Transit is used for JSON parsing." 
+  [message]
+  (letfn [(walk-fn [e]
+            (if (and (string? e)
+                     (first e)
+                     (contains? #{"~" "^" "`"} (first e))
```

Comment:
Please extract this set of with characters which should be escaped to a constant.